packaging/linux/deb/openra.lintian-overrides

```diff
@@ -0,0 +1,3 @@
+openra: duplicate-font-file usr/lib/openra/mods/common/FreeSans.ttf also in fonts-freefont-ttf
+openra: duplicate-font-file usr/lib/openra/mods/common/FreeSansBold.ttf also in fonts-freefont-ttf
+openra: binary-without-manpage usr/games/openra-server
```

Comment:
They are, and they are not disabled but ignored for good/temporary reasons; lintian still lists them for me.

If you can confirm that it can find the fonts if they're not at that location and if you are willing to add the depend on `fonts-freefont-ttf`, then we can rid of the first two.
The third warning is discussed above, it will be unignored once the man-page is created (#12525).

I choose to go for the overrides because they document that things are left unclean for a specific reason. It might be a good idea to document the reason though.